config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_alloydbusers.alloydb.cnrm.cloud.google.com.yaml

```diff
@@ -0,0 +1,160 @@
+    singular: alloydbuser
+  preserveUnknownFields: false
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    - description: When 'True', the most recent reconcile of the resource succeeded
+      jsonPath: .status.conditions[?(@.type=='Ready')].status
+      name: Ready
+      type: string
+    - description: The reason for the value in 'Ready'
+      jsonPath: .status.conditions[?(@.type=='Ready')].reason
+      name: Status
+      type: string
+    - description: The last transition time for the value in 'Status'
+      jsonPath: .status.conditions[?(@.type=='Ready')].lastTransitionTime
+      name: Status Age
+      type: date
+    name: v1beta1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'apiVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              clusterRef:
+                oneOf:
+                - not:
+                    required:
+                    - external
+                  required:
+                  - name
+                - not:
+                    anyOf:
+                    - required:
+                      - name
+                    - required:
+                      - namespace
+                  required:
+                  - external
+                properties:
+                  external:
+                    description: 'Allowed value: The `name` field of an `AlloyDBCluster`
+                      resource.'
+                    type: string
+                  name:
+                    description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names'
+                    type: string
+                  namespace:
+                    description: 'Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/'
+                    type: string
+                type: object
+              databaseRoles:
+                description: List of database roles this database user has.
+                items:
+                  type: string
+                type: array
+              password:
```

Comment:
This looks to be a sensitive field, and should support value from similar to Sql User. Could you check if we forgot to mark it as sensitive in the TF code?

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/c52face3c667f180dc7e6c2e51eb031a64e7c275/crds/sql_v1beta1_sqluser.yaml#L111

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/c52face3c667f180dc7e6c2e51eb031a64e7c275/third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/services/sql/resource_sql_user.go#L91

Reply:
Thanks for pointing this out, somehow missed it. I think it should be treated as a sensitive field, will look into it

Reply:
Have updated the tf patch to mark "password" as a sensitive field.
Raised a separate PR for that. Will continue with this PR once that is merged
https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1020

Reply:
@nitinchugh-google : #1020 is merged, you can update this PR to pick up the change.